src/viewport/api/photo.py

```
@@ -145,100 +176,237 @@ async def batch_presigned_uploads(
+            media_type = MediaType.VIDEO if is_video else MediaType.IMAGE
+
+            if media_type == MediaType.IMAGE:
+                # --- Image: single PUT ---
+                try:
+                    presigned = s3_client.generate_presigned_put(
+                        object_key=object_key,
+                        content_type=content_type,
+                        content_length=file_size,
+                        expires_in=900,
+                    )
+                except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)